public/auth_end.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+ <head>
+   <script src="/MicrosoftTeams.min.js"></script>
```

Comment:
I don't see a copy of this file in the src folder. Were you planning to add it?

Reply:
I'm using a local copy right now, waiting for the SDK beta release, then I will update the link here